django/contrib/admin/static/admin/css/forms.css

```diff
@@ -37,17 +37,19 @@ label {
 
 /* RADIO BUTTONS */
 
-form ul.radiolist li {
-    list-style-type: none;
+form div.radiolist div {
+    overflow:visible;
+    display: inline-block;
```

Comment:
There may be specific reasons why introducing `overflow` and `inline-block` here make sense, but fwiw prior to moving to `<div>` they were `float: left` instead, in case that helps as a target for _"how things were"_

Moving to `<div> > <div>` also looks to have erased the following rules from being applied, which may have an effect in terms of spacing:
```
ul > li {
    ...
    padding: 1px 0;
}

li, dt, dd {
    font-size: 13px;
    line-height: 20px;
}
```